packages/new-polymath-shared/src/testUtils/factories.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  ProcedureTypes,
+  TransactionQueueStatus,
+  TransactionQueuePojo,
+} from '~/typing/types';
+
+export const transactionQueuePojo = (
+  data?: Partial<TransactionQueuePojo>
+): TransactionQueuePojo => {
+  return {
+    ...(data || {}),
```

Comment:
Shouldn't this spread be at the end of the object so that the provided data overrides the hardcoded properties?

Reply:
Urgh true, my bad